.github/workflows/ci.yml

```diff
@@ -99,7 +99,7 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.6", "3.7", "3.8", "3.9"]
+        python-version: ["3.7", "3.8", "3.9"]
```

Comment:
Nautobot is not dropping 3.6, I don't think we should either.

Reply:
That is fine, but I do want to explain the consequences of continuing support of 3.6. Support for newer type annotation style (i.e. using `list` instead of `typing.List`) only extends back to 3.7 with the `__future__.annotations` module. Using `typing.List` is deprecated, and will raise import errors (I believe this still works in 3.10, but I need to verify).

Reply:
fair enough, but I still think it makes sense to be in sync with nautobot.